runtime/src/assets/stylesheets/shared/_variables.less

```diff
@@ -0,0 +1,4 @@
+@font-path: '~patternfly/src/fonts/';
```

Comment:
Are these use anywhere? if not, they should be removed. They are not used anymore in fabric8-ui.